### DISCUSSION TITLE
Welcome to BiG-SCAPE Discussions!

### DISCUSSION THREAD

**Discussion by jorgecnavarrom** [medema-group/BiG-SCAPE#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by onazora** [comment]
Hi everyone! I hope that you are all doing great. I am sincerely in need of your help. I am a firsttime user of BiG-Scape and CORASON. For over a week now, I have been struggling to analyze my .gbk files and regardless of the platform used, my analysis never crossed the 0%hmm Search. Please what is really going on? Or better still, what am I doing wrong? Please if anyone could help me, I will greatly appreciate it. Thank you everyone.
